src/command.cpp

```diff
@@ -77,11 +77,26 @@ void Command::arg(const char* name, const char* value, int quotes)
     addcomma();
     json.append("\"");
     json.append(name);
-    json.append(quotes ? "\":\"" : "\":");
-    json.append(value);
     if (quotes)
     {
-        json.append("\"");
+        json.append("\":\"");
+        size_t valueLen = strlen(value);
+        for (size_t i = 0; i < valueLen; i++)
+        {
+            if (value[i] == '\"'
+                    || value[i] == '\\'
+                    || value[i] == '/')
```

Comment:
According to the JSON spec (https://www.ietf.org/rfc/rfc4627.txt, Strings section), we don't need to escape /, but we do need to escape any control codes in the string.

Reply:
Yep, according to the spec those are the chars that should be escaped for Strings:
```
RFC 4627                          JSON                         July 2006

         string = quotation-mark *char quotation-mark

         char = unescaped /
                escape (
                    %x22 /          ; "    quotation mark  U+0022
                    %x5C /          ; \    reverse solidus U+005C
                    %x2F /          ; /    solidus         U+002F
                    %x62 /          ; b    backspace       U+0008
                    %x66 /          ; f    form feed       U+000C
                    %x6E /          ; n    line feed       U+000A
                    %x72 /          ; r    carriage return U+000D
                    %x74 /          ; t    tab             U+0009
                    %x75 4HEXDIG )  ; uXXXX                U+XXXX
```
I only added the printable chars with the specific case we aim to solve in mind. But a proper escaping should include all the above, indeed.